### PR TITLE
making arguments to be optional for old send sms interface

### PR DESCRIPTION
**Pull request by huzaif-plivo**

### CODE REVIEW

**Review by narayana-plivo — CHANGES_REQUESTED** [review]
Please clean up commented code

File: src/Plivo/Resources/Message/MessageInterface.php [hunk 1]
```diff
@@ -211,6 +261,37 @@ public function __call($name_of_method, $arguments)
...
+        
+        elseif($name_of_method=='list')
+        {
+            if (empty($arguments[0]))
+            {
+                $args = [];
+            }
+            else
+            {
+                $args = $arguments[0];
```
Comment (narayana-plivo) [inline src/Plivo/Resources/Message/MessageInterface.php:274]: why one only arguments is considering on list?

Comment (huzaif-plivo) [reply src/Plivo/Resources/Message/MessageInterface.php:274]: @narayana-plivo we are receiving all the parameters in our first array index.
```
$response = $client->messages->list([ 'limit' => 5,'offset' => 0]);
```

Comment (narayana-plivo) [reply src/Plivo/Resources/Message/MessageInterface.php:274]: is argument type should be array ?

File: src/Plivo/Resources/Message/MessageInterface.php [hunk 2]
```diff
@@ -159,42 +160,91 @@ public function __call($name_of_method, $arguments)
...
+            }
+            
             $dst = $arguments[1];
-            $text = $arguments[2];
-            $optionalArgs = $arguments[3];
-            $powerpackUUID = $arguments[4];
+            
+            if (empty($arguments[2]) || is_null($arguments[2]))
+            {   
+                $text = null;
```
Comment (narayana-plivo) [inline src/Plivo/Resources/Message/MessageInterface.php:198]: what case it should be null
?

Comment (huzaif-plivo) [reply src/Plivo/Resources/Message/MessageInterface.php:198]: @narayana-plivo  If the type is mms, the text is treated as an optional parameter.

File: src/Plivo/Resources/Message/MessageInterface.php (hunk 1 above)
Comment (huzaif-plivo) [reply src/Plivo/Resources/Message/MessageInterface.php:274]: @narayana-plivo yeah, we are accepting argument as array, for both old & new messaging interface & List all messages.

**Pull request closed without merging**